graylog2-server/src/main/java/org/graylog2/plugin/inject/Graylog2Module.java

```diff
@@ -465,6 +466,14 @@ protected void registerJacksonSubtype(Class<?> klass, String name) {
         return Multibinder.newSetBinder(binder(), ConstraintChecker.class);
     }
 
+    protected Multibinder<PluginUISettingsProvider> pluginUISettingsProviderBinder() {
+        return Multibinder.newSetBinder(binder(), PluginUISettingsProvider.class);
+    }
+
+    protected void addPluginUISettingsProvider(Class<? extends PluginUISettingsProvider> uiSettingsProviderClass) {
```

Comment:
We usually add the plugin related module methods into `PluginModule`. Is there a specific reason to put the method into this file?

Reply:
It went here mostly due to me not knowing the right place to put it.  I will move it.